src/manifolds/FixedRankMatrices.jl

```diff
@@ -81,9 +81,9 @@ Base.:(==)(x::SVDMPoint, y::SVDMPoint) = (x.U == y.U) && (x.S == y.S) && (x.Vt =
 @doc raw"""
     UMVTVector <: TVector
 
-A tangent vector that can be described as a product $UMV^\mathrm{H}$, at least
-together with its base point, see for example [`FixedRankMatrices`](@ref). This
-vector structure stores the additionally (to the point) required fields.
+A tangent vector that can be described as a product $UMV^\mathrm{H} + U_p V^\mathrm{H} + U V_p^\mathrm{H}$,
```

Comment:
Just two notation remarks
* it is easier/better to use tow backticks (``) instead of the usual ($), since that is the default Markdown syntax and a little more Julia friendly
* Usually our points are ``p,q\in\mathcal M`` while tangents are ``X,Y\in T_p\mathcal M``, see https://juliamanifolds.github.io/Manifolds.jl/latest/misc/notation.html . I know this is sometimes challenging when they are all matrices, but I would prefer writing ``X= UMV...`` since that's the tangent vector and then ``p = U\SigmaV...``, where I think I used ``S`` instead of a ``\Sigma`` until now here. We can change that last point if you like, but then please consistently.

Reply:
I see. I used the notation `U_p` because the columns are perpendicular to the ones of `U` (same for `V_p` and `V`).
Since we have `p = U S V^T` it's a bit confusing to have the tangent vector represented by `(U, M, V^T)`.
Maybe `\tilde{U}`?

Reply:
Sure, we are a little different from a few low rank papers here, but we try to keep a unified notation over all our manifolds.

What do you think about specifzying both with indices, i.e. `p = U_pS_pV_p^T` is the point on the manifold and the three matrices for the condensed form of the tangent `X` are `U_X`, `M_X` and `V_X` ? Sure this should then also be adapted in the `FixedRankMatrices` doc string, I must have misread that doc string (thinking U_p and V_p belong to p and not that they are orthogonal) yesterday. But I think this only further demonstrates, that it has to be very clear which matrices belong to what element.

Reply:
Actually in the doc https://juliamanifolds.github.io/Manifolds.jl/stable/manifolds/fixedrankmatrices.html you are also using `U_p` and `V_p` for the tangent components.
I could change that to the notation you suggested above if you like.
What do you think?

Reply:
Yes, because at that time I was confused myself. I would be in favour of changing that then as well, yes.

Reply:
Oh ok, actually I misread what you said I think. All good! I'll make the changes.